TasksTracker.Processor.Backend.Svc/TasksTracker.Processor.Backend.Svc.csproj

```diff
@@ -14,5 +14,4 @@
     <PackageReference Include="SendGrid" Version="9.28.1" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
   </ItemGroup>
-
```

Comment:
Just to trigger build on merge as actions to publish image will run when there is change to the source code of the three services